src/main/resources/application.yaml

```diff
@@ -1,6 +1,14 @@
 server:
   port: ${PORT:4455}
 
+audit:
+  log:
+    enabled: true
+    ignore:
+      # http statuses with comma separated to ignore audit logs
+      statuses:
```

Comment:
What is the scenario where we would have audit logs disabled ?   Do we need to override any of this audit configuration in the charts files ?

Reply:
@sullivangareth 
going by the cpo-case-payment-orders-api example, which this implementation follows, the audit log is always enabled.  The Jira ticket never actually asked for the audit logging feature to be configurable this way.  I'm assuming this was requested by some other requirement.  Shall we take it out altogether if it's confusing?

Reply:
I think it's already there https://github.com/hmcts/ccd-case-document-am-api/pull/412/files#diff-2109c901004111239683f91813f776c081fc8ec9b94eda17d5dc3142cdc289c1R22
The default in the application.yaml is true. So, it's configurable. 
Whenever we want to disable we can always add an environment variable in the related flux config.